controllers/controller.games.js

```diff
@@ -1,4 +1,4 @@
-const {selectReviews,selectCategories,} = require('../models/model');
+const {selectReviews,selectCategories,selectReviewID} = require('../models/model');
```

Comment:
Really like the consistent naming pattern of the functions, you may want to split these models/controllers out to seperate files to keep your code maintainable as you move forward.